download.sh

```diff
@@ -51,5 +51,8 @@ download_if_not_exists ${MAVEN_MIRROR}/org/apache/kyuubi/${TPCH_CONNECTOR_JAR_NA
 ICEBERG_SPARK_JAR_NAME=iceberg-spark-runtime-${SPARK_BINARY_VERSION}_${SCALA_BINARY_VERSION}
 download_if_not_exists ${MAVEN_MIRROR}/org/apache/iceberg/${ICEBERG_SPARK_JAR_NAME}/${ICEBERG_VERSION}/${ICEBERG_SPARK_JAR_NAME}-${ICEBERG_VERSION}.jar
 
+HUDI_SPARK_BUNDLE_JAR_NAME=hudi-spark${SPARK_BINARY_VERSION}-bundle_${SCALA_BINARY_VERSION}
+download_if_not_exists https://github.com/yanghua/incubator-hudi/releases/download/release-${HUDI_VERSION}/${HUDI_SPARK_BUNDLE_JAR_NAME}-${HUDI_VERSION}.jar
```

Comment:
are they present at maven central?

Reply:
No, it seems `[hbase-server](https://mvnrepository.com/artifact/org.apache.hbase/hbase-server/2.4.9)` did not provide a hadoop3 version. Users should rebuild hbase from the source code [manually](https://hudi.apache.org/docs/troubleshooting#how-can-i-resolve-the-nosuchmethoderror-from-hbase-when-using-hudi-with-metadata-table-on-hdfs).

Reply:
https://issues.apache.org/jira/browse/HUDI-4341

Reply:
that's crazy... Spark has shipped Hadoop3 by default since 3.2.0

Reply:
> https://issues.apache.org/jira/browse/HUDI-4341

There is a discussion about this: https://github.com/apache/hudi/issues/5765#issuecomment-1169531895

Of couse, we have another way to ignore this, by setting `hoodie.metadata.enable=false`. However, this is a limitation for users who want to explore Hudi's strong features.

Reply:
> that's crazy... Spark has shipped Hadoop3 by default since 3.2.0

Everything has been packaged into the bundle. So it is clear and can be controlled.

Reply:
let's add a comment before this download link to explain the reason